test/evergreen/code_complexity_analysis.py

```diff
@@ -0,0 +1,118 @@
+# importing the module
+import argparse
+import json
+import csv
+import os.path
+import ast
+from heapq import nlargest
+import pandas as pd 
+from collections import defaultdict
+    
+class CodeComplexity:
+    dataFrame = pd.DataFrame()
+    topNRegions = 5
```

Comment:
Should they go to the __init__ method?

Reply:
Fixed in 7cc186a02197e388c9a2fcaa97331d24c84376a5